config/nvim/plugins/pomo.lua

```
@@ -21,6 +21,151 @@ function pomo.config()
+							start = function()
+								create_floating_window()
+							end,
+							tick = function(time_left)
+								update_window(time_left)
+							end,
+							done = function()
+								close_window()
+
+								-- Show completion message
+								local completion_buf = vim.api.nvim_create_buf(false, true)
+								local completion_lines = {}
```

Luacheck (1.2.0)

[warning] value assigned to variable 'completion_lines' is unused

(W311)